docs/architecture-docs.md

```diff
@@ -0,0 +1,29 @@
+# Ground Android app architecture
+
+The Ground Android app is written in Java using the [Android Jetpack](https://developer.android.com/jetpack) and [RxJava](https://github.com/ReactiveX/RxJava) libraries. This page provides an overview of the high-level app architecture, including key components and libraries.
```

Comment:
It relies heavily on these two, but they don't form an exhaustive list of libraries used; can this be reworded somehow to make that more clear?

Reply:
Done.